packages/forms-web-app/src/controllers/projects/representations.js

```diff
@@ -62,7 +65,10 @@ exports.getRepresentation = async (req, res) => {
 			RepFrom: representation.data.RepFrom,
 			RepresentationRedacted: representation.data.RepresentationRedacted,
 			DateRrepReceived: representation.data.DateRrepReceived,
-			attachments: representation.data.attachments
+			attachments: representation.data.attachments,
+			hideAllExaminationDocumentsLink: true,
+			hideRecommendationAndDecisionLink: true,
+			hideExaminationTimetableLink: true
 		});
```

Comment:
Hey @mauriciokin Would it maybe be an idea to add these 3 variables (if they always have the same values) to a config file which is imported by each of the controllers that use them?

Reply:
very good shout! fixed it!